test/tools/libtesteth/TestHelper.cpp

```diff
@@ -477,7 +422,9 @@ void executeTests(const string& _name, const string& _testPathAppendix, const st
 	}
 	try
 	{
-		cnote << "TEST " << name << ":";
+		if ((Options::get().singleTest && Options::get().singleTestName == name) || !Options::get().singleTest)
+			cnote << "TEST " << name << ":";
```

Comment:
so that when you use --statediff you would see only the name of the test you are running with --singletest option. 
other names make it hard to see the changeLog in console when there are many tests